elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -113,6 +114,33 @@ public void commit() {
         return list;
     }
 
+    /**
+     * Internal loadObjects helper - loads pagination objects
+     * @param loadClass The class Type reference
+     * @param criterion The optional Criterion
+     * @param pagination The Pagination object
+     * @param <T> The class type
+     * @return The ScrollableIterator
+     */
+    public <T> Iterable<T> loadObjects(Class<T> loadClass, Criterion criterion, Pagination pagination) {
```

Comment:
I think it makes sense (given Bill's join work) to only worry about pagination inside the root collection (and not the includes)).  Basically, I agree with only adding support for now at loadObjects.